snforge_std/src/cheatcodes/tx_info.cairo

```diff
@@ -3,6 +3,11 @@ use starknet::info::v2::ResourceBounds;
 use snforge_std::cheatcodes::{CheatSpan, CheatTarget, validate_cheat_target_and_span};
 use super::super::_cheatcode::handle_cheatcode;
 
+
+/// A structure used for setting individual fields in `TxInfo`
+/// All fields are optional, with optional value meaning as defined:
+/// - `None` means that the field is going to be reset to default value
+/// - `Some(n)` means that the value will be set to the `n` value
 #[derive(Copy, Drop, Serde)]
 struct TxInfoMock {
```

Comment:
Should this be commented too? Not individual fields, but the struct overall

Reply:
The struct is commented, look above